.github/workflows/aws_ecr.yml

```diff
@@ -77,10 +91,85 @@ jobs:
             type=semver,pattern={{major}}.{{minor}}
 
       - name: Build and push Docker images
+        if: github.event.name != 'pull_request_review'
         uses: docker/build-push-action@v4
         with:
           context: ${{ matrix.context }}
           file: ${{ matrix.file }}
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+  deploy:
+    name: Deploy to EKS
+    needs: build
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write # This is required for requesting the JWT
+    if: github.event.review.state == 'approved' || github.event_name == 'push'
+
+    env:
+      projects: |
+        governance-ga-agent
+        governance-multitenant-agent
+        governance-multitenant-web
+        governance-trust-registry
+        governance-webhooks-web
+        governance-endorser
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Checkout Helm chart
+        uses: actions/checkout@v3
+        with:
+          repository: didx-xyz/cloud_api_helm
+          token: ${{ secrets.HELM_REPO_TOKEN }} # private access token here, for HTTPS clones
+          path: helm/Chart
+          ref: CloudAPI-Environment-Variable-Changes
```

Comment:
This needs to be removed once https://github.com/didx-xyz/cloud_api_helm/pull/2 is merged